static/js/compose.js

```diff
@@ -565,7 +527,6 @@ function send_message(request) {
     if (request === undefined) {
         request = create_message_object();
     }
-    exports.snapshot_message(request);
```

Comment:
The only case this might be useful is when the message sending fails. If that happens, then the message doesn't disappear right? You get a dummy message on the frontend with an option to retry sending. So in that case, saving to drafts isn't helpful.